client/src/buttons/LikeIcon.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function LikeIcon({ onThumbsUp }) {
+  return (
+    <button
```

Comment:
What does this button do? How can a non-sighted user find out? When you ran Lighthouse, what did the report say and what steps did you take to address it?

Reply:
aria-label added to the like button.

Reply:
Hi Sally, thank you so much for reviewing my code, I really appreciate your effort.